src/essreflectometry/amor/__init__.py

```diff
@@ -5,17 +5,9 @@
 
 import scipp as sc
 
-from .. import calibrations, normalize
 from .. import providers as reflectometry_providers
-from ..types import (
-    Alpha,
-    BeamSize,
-    CriticalEdge,
-    DetectorSpatialResolution,
-    Gravity,
-    MValue,
-    SampleSize,
-)
+from ..supermirror.types import Alpha, CriticalEdge, MValue
```

Comment:
Why import contents here? Then we lose the "namespace" again.

Reply:
They are imported to define default parameters for the Amor example.
But we could import only `supermirror` and use them like `supermirror.types.Alpha` instead to keep the namespace.